SocialFilter/sections/specific.txt

```diff
@@ -347,6 +347,7 @@ newsd.in##.followusgoogle
 animationxpress.com##.widget_simple_facebook_page_feed_widget
 testsigma.com##.share_blog
 maximuzz.ru##.full-track__share
+newzit.com##div[class^="share"]
```

Comment:
Also remove it
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/141772692-6cb9c967-b3dd-4e03-b919-5921f15dabbd.png)

</details><br/>